Userland/Libraries/LibWeb/HTML/HTMLOutputElement.cpp

```diff
@@ -25,6 +26,29 @@ void HTMLOutputElement::initialize(JS::Realm& realm)
     WEB_SET_PROTOTYPE_FOR_INTERFACE(HTMLOutputElement);
 }
 
+void HTMLOutputElement::visit_edges(Cell::Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+    visitor.visit(m_html_for);
+}
+
+void HTMLOutputElement::form_associated_element_attribute_changed(FlyString const& name, Optional<String> const& value)
+{
+    if (name == HTML::AttributeNames::for_) {
+        if (m_html_for)
+            m_html_for->associated_attribute_changed(value.value_or(String {}));
+    }
+}
+
+// https://html.spec.whatwg.org/multipage/form-elements.html#dom-output-htmlfor
+JS::GCPtr<DOM::DOMTokenList> HTMLOutputElement::html_for()
```

Comment:
Oh sorry, I missed this (tiny) nit thing earlier, but this should return a `NonullGCPtr`